Changed sankeys content to pathways (#29229)
DataDog/documentation: content/en/product_analytics/segmentation/_index.md

## Leverage segments across Product Analytics

### In pathways

In the Pathway page, you can filter the data in the visualization to reflect a selected segment of users. This allows you to look at the experience and traffic patterns of a particular set of users in a given segment. The below example shows a Pathway diagram only for users in the "Premium Shopist Customers" segment.

{{< img src="product_analytics/segmentation/segmentation-pathways.png" alt="Filter down the Pathway visualization to reflect a selected segment of users.">}}

### In analytics explorer

> You can filter the data in the Analytics Explorer to reflect a selected set of users that are in a given segment. The example below shows a list of users in the "Premium Shopist Customers" segment who were active in the last month, organized by the number of sessions they had.

Reviewer: ```suggestion
You can filter the data in the Analytics Explorer to reflect a selected set of users that are in a given segment. The example below shows a list of users in the "Premium Shopist Customers" segment who were active in the last month, organized by the total number of events.
```

> {{< img src="product_analytics/segmentation/analytics-explorer.png" alt="Show a list of users in the Premium users segment who were active in the last month, organized by the number of sessions they had.">}}

Reviewer: ```suggestion
{{< img src="product_analytics/segmentation/analytics-explorer.png" alt="Show a list of users in the Premium Shopist Customers segment who were active in the last month, organized by the number of sessions they had.">}}
```

Reviewer: It looks like the new graph is organized by the number of `All events` rather than `all sessions` as in the previous screenshot. Should this be updated to say "organized by the total number of events"?

Author: Great question, I'm looking into this `All events` vs `all sessions` and will follow-up soon!

Author: Hi @buraizu, I've confirmed that this is ok to add for as `All events` for now

Reviewer: @iadjivon thanks for confirming! I've updated my suggestion to correct the alt text as mentioned.
